package.json

```
@@ -30,28 +30,28 @@
   },
   "scripts": {
     "start": "start-storybook -p 9001 -c storybook -s ./storybook/static",
-    "build": "lerna run --parallel build -- -- --env production && npm run ts:emit",
+    "build": "lerna run --parallel build -- -- --env production && yarn ts:emit",
```

Let's update `lerna.json` so that it uses `yarn` when issuing `run` commands. Docs: https://github.com/lerna/lerna/tree/master/commands/run#--npm-client-client

Then we can update this to: 
```suggestion
    "build": "lerna run --parallel build -- --env production && yarn ts:emit",
```

Still need that other `--` to pass it to webpack tho.

---

👍